### PR TITLE
feat: add "related sessions" in exhibitors

### PR DESCRIPTION
**Pull request by sachinchauhan2889**
Add migrations for https://github.com/fossasia/open-event-frontend/issues/6864

#### Checklist

- [ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [ ] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by sachinchauhan2889** [comment]
@iamareebjamal  sir getting this error while creating migration

**Comment by iamareebjamal** [comment]
Go to `.venv/lib/python3.8/site-packages/graphene_sqlalchemy/types.py`

Line 210

Comment out:

```
# assert is_mapped_class(model), (
#     "You need to pass a valid SQLAlchemy Model in " '{}.Meta, received "{}".'
# ).format(cls.__name__, model)
```

Run migrate command, fix the error

File: app/models/exhibitor.py [hunk 1]
```diff
@@ -4,6 +4,13 @@
 from app.models.helpers.timestamp import Timestamp
 from app.models.helpers.versioning import clean_html, clean_up_string
 
+exhibitors_sessions = db.Table(
+    'exhibitors_sessions',
+    db.Column('session_id', db.Integer, db.ForeignKey('session.id', ondelete='CASCADE')),
```
Comment (iamareebjamal) [inline app/models/exhibitor.py:9]: There is no table named `session`

Comment (sachinchauhan2889) [reply app/models/exhibitor.py:9]: yes sir. this fixes the error.

**Comment by iamareebjamal** [comment]
sessions.py API remaining

**Pull request merged**